File: Auction/Auction/DTO/ItemDTO.cs
namespace Auction.DTO
{
    public class ItemDTO
    {

        public int Id { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public float Price { get; set; }

        public bool IsAvailable { get; set; } = true;

        public int CategoryId { get; set; }

        public string CategoryName { get; set; }

    }
}

File: BidManager/src/main/java/pt/upskill/bidmanager/DTO/ItemDTO.java
package pt.upskill.bidmanager.DTO;


import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ItemDTO {
    public int id;

    public String name;

    public String description;

    public float price;

    public Boolean IsAvailable;

    public int CategoryId;

    public String categoryName;
}
